doc/source/examples/networktheory/Renormalizing S-parameters.ipynb

```diff
@@ -122,61 +111,62 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You could also renormalize to a complex port impedance if you're crazy"
+    "## Complex Impedances "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You could also renormalize to a complex port impedance if you're crazy. For example, renormalizing to 50j, one would expect:\n",
+    "$$\n",
+    "\\Gamma^{'} = \\frac{50-50j}{50+50j} = 50\\frac{1-j}{1+j} = -50j\n",
+    "$$\n",
+    "\n",
+    "However, one finds an unexpected result when plotting the Smith chart:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "match_at_50 = rf.wr10.match()\n",
-    "match_at_50.renormalize(50j)\n",
-    "match_at_50.plot_s_smith(**kw)"
+    "match_at_50.renormalize(50j)  # same as renormalize(50j, s_def='power')\n",
+    "match_at_50.plot_s_smith(**kw)  # expect -1j"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Complex Impedances "
+    "This is because the default behaviour of scikit-rf is to use _power-waves_ scattering parameter definition (since it is the most popular one is CAD softwares). But the _power-waves_ definition is [known to fail in such a case](https://www.nist.gov/system/files/documents/2017/05/09/MicrowaveCircuitTheory-proof.pdf). This is why scikit-rf also implement the _pseudo-waves_ scattering parameters definition, but you have to specify it using the `s_def` parameter:  "
```

Comment:
great explanation and ref link.

Reply:
Thanks. Also learned a lot, for example that few textbooks are very bad at explaining this, or even mix up in the definitions... Even commercial software helps are not very clear about what is really implemented...